docs/DeveloperGuide.md

```diff
@@ -430,7 +430,20 @@ The following activity diagram summarizes what happens when a user executes a ne
 - Modify the format of `DATE` to be `d-M-yyyy` and `DATETIME` to be `d-M-yyyy`T`H:mm` instead.
 - This allows FINDvisor to accept both single and double-digits day, month and hour values as valid `DATE` and `DATETIME` values and would not require users to pad these single digit values with a leading zero.
 
-### 5. Show warning to user when scheduling an overlapping meeting
+### 5. Notfiy user if data file is invalid
+**Current Implementation**: 
+
+- No error message is displayed to the user when this occurs and users are not informed of the data loss.
+- If new state changing command is made when the data file is invalid, the data file will be overwritten.
+
+**Proposed Enhancement**: 
+
+- Display a message in the Command Result Box if the data file cannot be parsed on startup.
```

Comment:
```suggestion
- Display a message in the *Command Result Box* if the data file cannot be parsed on startup.
```
I think it is better to make it italics, given that it is how we will reference our UI components.